linux_os/guide/services/ssh/ssh_server/sshd_set_loglevel_verbose/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+title: 'Set SSH Daemon LogLevel to VERBOSE'
+
+description: |-
+    The VERBOSE parameter specifices that record login and logout activity will be logged.
+    To specify the log level in
+    SSH, add or correct the following line in the <tt>/etc/ssh/sshd_config</tt> file:
+    <pre>LogLevel VERBOSE</pre>
```

Comment:
Logging user login/logout is default behavior for most Linux distributions, and setting VERBOSE is not required. For example on RHEL, CentOS, or Scientific Linux:

`````
Apr 17 22:19:09 rhel7-devbox sshd[10249]: Accepted password for shawn from 10.211.55.2 port 50597 ssh2
Apr 17 22:19:09 rhel7-devbox sshd[10249]: pam_unix(sshd:session): session opened for user shawn by (uid=0)
.....
pr 17 22:20:18 rhel7-devbox sshd[10253]: Received disconnect from 10.211.55.2 port 50597:11: disconnected by user
Apr 17 22:20:18 rhel7-devbox sshd[10253]: Disconnected from 10.211.55.2 port 50597
Apr 17 22:20:18 rhel7-devbox sshd[10249]: pam_unix(sshd:session): session closed for user shawn
`````

Could/should the description be modified to state something like "If your Linux distribution's OpenSSH Server does not capture user login/logouts, LogLevel VERBOSE may be needed"?

Reply:
SLES also is verbose enough by default. I guess the worry was that it might be explicitly disabled by configuring `ERROR/FATAL/QUIET`. Our STIG strictly mandates `VERBOSE` to prevent that.

I can adjust the wording to capture that.